cg/apps/gt.py

```diff
@@ -37,3 +46,31 @@ def upload(self, bcf_path: str, samples_sex: dict, force: bool=False):
                 analysis_obj.sex = samples_sex[analysis_obj.sample_id]['analysis']
                 analysis_obj.sample.sex = samples_sex[analysis_obj.sample_id]['pedigree']
                 self.commit()
+
+    def get_trending(self, sample_id: str = '', days: str = '') -> dict:
+        """Get trending object with one or many samples."""
+        trending_call = self.base_call[:]
+
+        if sample_id:
+            trending_call.extend(['prepare-trending', '-s', sample_id])
+        elif days:
+            trending_call.extend(['prepare-trending', '-d', days])
```

Comment:
Is it ok to have no sample and no days?

Reply:
Nope. How should I handle it differently?